.github/workflows/scale-test-100-gce.yaml

```diff
@@ -124,6 +124,7 @@ jobs:
             --set=k8sServicePort=443 \
             --set=kubeProxyReplacement=true \
             --set=operator.replicas=1 \
+            --set=updateStrategy.rollingUpdate.maxUnavailable=100% \
```

Comment:
That looks a bit aggressive to me, considering the idea of making the number of nodes configurable. How about maybe picking a fixed number, or a lower percentage?

Reply:
Fair enough :+1: